packages/sdk/src/sdk/api/users/types.ts

```diff
@@ -4,16 +4,53 @@ import { ImageFile } from '../../types/File'
 import { HashId } from '../../types/HashId'
 import { getReaction, reactionsMap } from '../../utils/reactionsMap'
 
+export const UserEventsSchema = z.object({
+  referrer: z.optional(HashId),
+  isMobileUser: z.optional(z.boolean())
+})
+
+export const CreateUserSchema = z.object({
+  profilePictureFile: z.optional(ImageFile),
+  coverArtFile: z.optional(ImageFile),
+  onProgress: z.optional(z.function().args(z.number())),
```

Comment:
I realized zod only checks the arguments when the function is run (which makes sense, because i don't think there is a good way to check the argument types of a function in js) 


Anyway I changed stuff to this in my PR:
```suggestion
  onProgress: z.optional(z.function()),
```

But we can probably adjust when things get merged, there's some other changes to the onProgress type that will require a change anyway

Reply:
I'm actually not sure why the user stuff even has `onProgress` calls besides the fact that we upload artwork. But we don't actually _use_ it in the calling code. So I'm in favor of updating these later on.